### DISCUSSION TITLE
What is the difference between Authentication and  Authorization

### DISCUSSION THREAD

**Discussion by jkumar0021** [javax4u/interview-questions#37] [Q&A] [answered]

**Comment by javax4u** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/2442530/140606841-028c9f2a-9775-4139-a176-1457a929b416.png)
---
![image](https://user-images.githubusercontent.com/2442530/140606504-977a1478-8caa-4eb1-a123-05c17deb931a.png)

In English to remember which comes first **Auth**entication or **Auth**orization,  sort it alphabetically.  **E** comes before **O** so _Authentication_ will come before _Authorization_  मतलब पहले अधिकार फिर विशेषाधिकार  :)
